agent/src/apiclient.rs

```diff
@@ -150,7 +150,8 @@ async fn invoke_apiclient(args: Vec<String>) -> Result<Output> {
                             _ => {
                                 // API response was a non-transient error, bail out
                                 return apiclient_error::BadHttpResponseSnafu {
-                                    statuscode: error_statuscode,
+                                    args: args,
```

Comment:
Might be good to leave the status code there, which would make it easier to write code to handle the error elsewhere.

Reply:
+1 Agree with Sean. I think it'd make sense to have all 3:
- the command /args that were invoked
- the error status code
- the error contents